rpcs3/Emu/SysCalls/Modules/cellResc.cpp

```diff
@@ -344,14 +344,17 @@ s32 CalculateSurfaceByteSize(u32 mode, CellRescDsts *dsts)
 
 s32 CalculateMaxColorBuffersSize()
 {
-	s32 oneBufSize, bufNum, totalBufSize, maxBufSize;
+	s32 index, oneBufSize, bufNum, totalBufSize, maxBufSize;
 	maxBufSize = 0;
 
 	for (u32 bufMode = CELL_RESC_720x480; bufMode <= CELL_RESC_1920x1080; bufMode <<= 1)
 	{
 		if (s_rescInternalInstance->m_initConfig.supportModes & bufMode) 
 		{
-			oneBufSize   = CalculateSurfaceByteSize(bufMode, &(s_rescInternalInstance->m_rescDsts[GetRescDestsIndex(bufMode)]));
+			if ((index = GetRescDestsIndex(bufMode)) != 0)
+			{
+				oneBufSize = CalculateSurfaceByteSize(bufMode, &(s_rescInternalInstance->m_rescDsts[index]));
+			}
```

Comment:
1) There shouldn't be any defect there, from my understanding.
2) Fix doesn't make sense